open-ce/build_tree.py

```diff
@@ -54,27 +54,6 @@ def __init__(self,
         if self.build_command_dependencies is None:
             self.build_command_dependencies = []
 
-    def feedstock_args(self):
```

Comment:
This will still be used by the OpenShift implementation of Open-CE, so we should keep this function in here. Might need to add a test for it now that it won't be part of the normal usage of `build env`

Reply:
I'll put it back and look into adding a test.

Reply:
Done.